rust/Cargo.toml

```diff
@@ -13,7 +13,7 @@ server = []
 [dependencies]
 prost = "0.13.3"
 prost-types = "0.13.3"
-tonic = "0.12.3"
+tonic = { version = "0.12.2", features = ["tls"] }
```

Comment:
Did you mean to downgrade tonic to 12.2? There is a bunch of automation pushing this to 12.3.

Reply:
Thank you! I'm starting getting tired...